### PR TITLE
fix typos and update sentence text

### PR DESCRIPTION
**Pull request by mudssrali**
## What kind of change does this PR introduce?

docs update

## What is the current behavior?

A few grammar mistakes and typos, also not sure what lang (en-US or en-GB) you guys are following to keep the documentation, I just sensed, most of the documentation is following the en-GB rule, so I change the text accordingly.   

## What is the new behavior?

Feel free to include screenshots if it includes visual changes.

## Additional context

No, there's no additional context.

### CODE REVIEW

File: web/docs/guides/auth.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ Supabase makes it simple to manage your users.
 <source src="/videos/auth-zoom2.mp4" type="video/mp4" muted playsInline />
 </video>
 
-When a user signs up, Supabase assigns them a unique ID. You can reference this ID anywhere in your database. For example, you might create a `profiles` table referencing the user using a `user_id` field.
+When users sign up, Supabase assign them a unique ID. You can reference this ID anywhere in your database. For example, you might create a `profiles` table referencing the user using a `user_id` field.
```
Comment (kiwicopple) [inline web/docs/guides/auth.mdx:15]: > Supabase assign them

assigns?

File: www/components/BuiltWithSupabase/index.tsx [hunk 2]
```diff
@@ -16,7 +16,7 @@ const BuiltExamples = () => {
       <div className="text-center">
         <Typography.Title level={2}>What can you build with supabase?</Typography.Title>
         <Typography.Text>
-          <p className="lg:text-lg">There's many example apps and starter projects to get going</p>
+          <p className="lg:text-lg">There're many example apps and starter projects to get going</p>
```
Comment (kiwicopple) [inline www/components/BuiltWithSupabase/index.tsx:19]: Nice one. Perhaps we should remove the conjunction here for readability. "There're" is a bit awkward

**Comment by kiwicopple** [comment]
@mudssrali, I love contributions like this. Thanks for taking the time to go through the docs and improve them.

> not sure what lang (en-US or en-GB)

Yeah need to decide on this. I think we should stick with en-US, as that's where most of our users are and it fits better with some of the technical terminology (`color` not `colour`, `materliaize` not `materialise`)

File: web/docs/guides/auth.mdx (hunk 1 above)
Comment (mudssrali) [reply web/docs/guides/auth.mdx:15]: If the orga­ni­za­tion is referred to as “they” or “who,” use a plural verb with the company name. If the orga­ni­za­tion is referred to as “it” or “which,” use a singular verb. I think most of the organization takes plural.

File: www/components/BuiltWithSupabase/index.tsx (hunk 2 above)
Comment (mudssrali) [reply www/components/BuiltWithSupabase/index.tsx:19]: Yes, I think so.

**Comment by mudssrali** [comment]
> @mudssrali, I love contributions like this. Thanks for taking the time to go through the docs and improve them.
> 
> > not sure what lang (en-US or en-GB)
> 
> Yeah need to decide on this. I think we should stick with en-US, as that's where most of our users are and it fits better with some of the technical terminology (`color` not `colour`, `materliaize` not `materialise`)

So as you described(`color` not `colour`, `materliaize` not `materialise`), we should go for `en-US` for documentation if @awalias  and @steve-chavez  are also agreed to do so.

**Comment by mudssrali** [comment]
> > @mudssrali, I love contributions like this. Thanks for taking the time to go through the docs and improve them.
> > > not sure what lang (en-US or en-GB)
> > 
> > 
> > Yeah need to decide on this. I think we should stick with en-US, as that's where most of our users are and it fits better with some of the technical terminology (`color` not `colour`, `materliaize` not `materialise`)
> 
> So as you described(`color` not `colour`, `materliaize` not `materialise`), we should go for `en-US` for documentation if @awalias and @steve-chavez are also agreed to do so.

After this PR, will work on documentation to make it aligned with `en-US`.

**Comment by awalias** [comment]
`en-US` is fine with me, ya'll

**Comment by steve-chavez** [comment]
`en-us` also LGTM!

**Comment by mudssrali** [comment]
> @mudssrali, I love contributions like this. Thanks for taking the time to go through the docs and improve them.
> 
> > not sure what lang (en-US or en-GB)
> 
> Yeah need to decide on this. I think we should stick with en-US, as that's where most of our users are and it fits better with some of the technical terminology (`color` not `colour`, `materliaize` not `materialise`)

Hi @kiwicopple  I just change docs to `en-US` where possible, and also fix a more typos. Now you can review and merge. Thanks

**Comment by kiwicopple** [comment]
woo! thanks @mudssrali . this is great

**Pull request merged**